app/src/main/java/org/selfconference/android/speakers/SpeakerTypeAdapter.java

```diff
@@ -42,10 +42,11 @@
           builder.name(in.nextString());
           break;
         case KEY_BIO:
-          builder.bio(in.nextString());
+          String bio = parseNullableString(in);
+          builder.bio(Optional.fromNullable(bio).or(""));
           break;
         case KEY_PHOTO:
-          String url = parsePhoto(in);
+          String url = parseNullableString(in);
           builder.photo(Optional.fromNullable(url).or("http://google.com"));
```

Comment:
Yeah, I think it's the `http://google.com` URL. I think it would probably make sense to have a placeholder image URL here at some point, which can be downloaded, cached, and used throughout the app by Picasso instead of this weird URL that just prevents the app from crashing haha.